Orange/data/table.py

```diff
@@ -11,7 +11,6 @@
 from tempfile import NamedTemporaryFile
 from urllib.parse import urlparse, unquote as urlunquote
 from urllib.request import urlopen
-from urllib.error import URLError
```

Comment:
But it's unused (caught outside `from_url` in `OWFile`). 
Actually with `from .instance import *` it's a mess here. This import shadows `chain`, `Real` and `Integral` and imports `numpy`.